From e5850d435db26953ae202d88827d894935bec3a9 Mon Sep 17 00:00:00 2001
From: "Valery V. Vorotyntsev" <valery_vorotyntsev@xyratex.com>
Date: Wed, 25 Jul 2012 14:19:12 +0300
Subject: [PATCH 266/311] confc.h: rename filesystem_open_async()

- `_async' suffix is misleading, since the function itself is synchronous.
  s/filesystem_open_async/filesystem_open1/
  s/filesystem_open_sync/filesystem_open2/
  Thanks to Anatoliy for pointing this out!
- confc.[hc]: Straighten documentation of sm_filter() functions.
---
 conf/confc.c | 2 +-
 conf/confc.h | 8 +++++---
 2 files changed, 6 insertions(+), 4 deletions(-)

diff --git a/conf/confc.c b/conf/confc.c
index bbcab25..4db830a 100644
--- a/conf/confc.c
+++ b/conf/confc.c
@@ -465,7 +465,7 @@ struct sm_waiter {
 	struct c2_clink     w_clink;
 };
 
-/** Filters out intermediate states of c2_confc_ctx::fc_mach. */
+/** Filters out intermediate state transitions of c2_confc_ctx::fc_mach. */
 static bool sm_filter(struct c2_clink *link)
 {
 	return !c2_confc_ctx_is_completed(&container_of(link, struct sm_waiter,
diff --git a/conf/confc.h b/conf/confc.h
index 224b9fa..1405051 100644
--- a/conf/confc.h
+++ b/conf/confc.h
@@ -176,7 +176,8 @@ struct c2_mutex;
  * static void sm_waiter_init(struct sm_waiter *w, struct c2_confc *confc);
  * static void sm_waiter_fini(struct sm_waiter *w);
  *
- * static int filesystem_open_async(struct c2_conf_filesystem **fs)
+ * // Uses asynchronous c2_confc_open() to get filesystem configuration.
+ * static int filesystem_open1(struct c2_conf_filesystem **fs)
  * {
  *         struct sm_waiter w;
  *         int              rc;
@@ -198,13 +199,14 @@ struct c2_mutex;
  *         return rc;
  * }
  *
- * static int filesystem_open_sync(struct c2_conf_filesystem **fs)
+ * // Uses synchronous c2_confc_open_sync() to get filesystem configuration.
+ * static int filesystem_open2(struct c2_conf_filesystem **fs)
  * {
  *         return c2_confc_open_sync(fs, g_confc->cc_root,
  *                                   C2_CONF_BUF_STR("filesystem"));
  * }
  *
- * // Filters out intermediate state of c2_confc_ctx::fc_mach.
+ * // Filters out intermediate state transitions of c2_confc_ctx::fc_mach.
  * static bool sm_filter(struct c2_clink *link)
  * {
  *         return !c2_confc_ctx_is_completed(&container_of(link,
-- 
1.8.3.2

